docs/developer-guide/actions/writing-playbooks.rst

```diff
@@ -149,39 +149,39 @@ Every Robusta playbook has a list of :ref:`Triggers` and a list of :ref:`Actions
      - resource_babysitter: {}
      - add_deployment_lines_to_grafana: {}
 
-For every incoming event, Robusta will run any playbook, that *one* of it's ``triggers`` is matched.
+For every incoming event, Robusta will run actions if any of the ``triggers`` match.
 
 All the playbook's ``actions`` will be executed, according to the order specified in the configuration.
 
-Playbooks execution order is according to the order specified in the configuration as well.
+Playbooks execution order, is the same as the configuration file order. For example:
 
-There are few mechanisms in place, to stop the processing flow, if needed:
-
-1. Stop handing the current event. No other ``playbooks`` or ``actions`` will be executed
+.. code-block:: yaml
 
-.. code-block:: python
-   :emphasize-lines: 5
+   - triggers:  # playbook A
+     - on_deployment_create: {}
+     actions:
+     - resource_babysitter: {}
+   - triggers:  # playbook B
+     - on_deployment_create: {}
+     actions:
+     - add_deployment_lines_to_grafana: {}
 
-    @action
-    def event_report(event: EventChangeEvent, action_params: EventErrorReportParams):
+On the example above, ``playbook A`` will run before ``playbook B``
 
-       if DONT RUN ANYTHING ELSE ON THIS EVENT:
-           event.stop_processing = True  # no need to run any other playbook or action
-           return
+There is a mechanisms in place, to stop the processing flow, if needed:
 
-2. For a single ``playbook``, skip the following ``actions``. For this we will set the ``skip_playbook`` attribute
+Set ``event.stop_processing = True``. No other ``playbooks`` or ``actions`` will be executed
 
 .. code-block:: python
    :emphasize-lines: 5
 
     @action
     def event_report(event: EventChangeEvent, action_params: EventErrorReportParams):
```

Comment:
I would give the function here the same name as one of the actions above. (Maybe just name them action1, action2 etc)

Reply:
fixed